app/controllers/search_controller.rb

```diff
@@ -104,6 +106,7 @@ def user_params
   end
 
   def feed_params
+    params[:tag_names] = params[:tag_names].split(",") if params[:tag_names]
```

Comment:
Ideally, we need to fix this issue in the javascript(`new URLSearchParams(dataHash).toString()`) but since this is the only search param with multiple values I took the easy way out(for me) and did this in Ruby.